06_neighborhoods/Transportation/transit_trips_cost_county_subgroups.R

```diff
@@ -0,0 +1,645 @@
+############################################################
+
+# SUBGROUP FILE
+
+# Process for SUBGROUP-level:
```

Comment:
This script does not load any packages. I believe you need at a minimum tidyverse and tidycensus.